torchaudio/io/_stream_reader.py

```diff
@@ -756,7 +778,13 @@ def pop_chunks(self) -> Tuple[Optional[torch.Tensor]]:
                 Buffer contents.
                 If a buffer does not contain any frame, then `None` is returned instead.
         """
-        return self._be.pop_chunks()
+        ret = []
+        for chunk in self._be.pop_chunks():
+            if chunk is None:
+                ret.append(None)
+            else:
+                ret.append(_ChunkTensor(chunk[0], chunk[1]))
+        return ret
```

Comment:
should update the return type annotation and docstring

Reply:
Since documenting the class means we need to make the class public. I made couple of changes. (making it inner class of StreamReader, decorated dataclass). Can you take a look again?